src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form big-form"
+      data-qa="big"
+    >
+      <label class="form__label">
+        <input
+          class="form__input form__input--big-imput"
+          type="text"
+          data-qa="keypress"
+          onfocus="this.placeholder=''"
+          onblur="this.placeholder='Try &#34;Los Angeles&#34;'"
```

Comment:
In this code, onfocus and onblur are JavaScript events.
According to Requirements we don't have to use JavaScript